api/rest/restcore/users_rest.php

```diff
@@ -76,20 +106,37 @@ function rest_user_get_me( \Slim\Http\Request $p_request, \Slim\Http\Response $p
  * @return \Slim\Http\Response The augmented response.
  */
 function rest_user_get( \Slim\Http\Request $p_request, \Slim\Http\Response $p_response, array $p_args ) {
-	$t_user_id = $p_args['user_id'];
-	if( $t_user_id <= 0 ) {
-		return $p_response->withStatus( HTTP_STATUS_BAD_REQUEST, "Invalid user id $t_user_id" );
+	$t_user_ref = $p_args['ref'];
+
+	$t_select = $p_request->getParam( 'select', null );
+	if( is_null( $t_select ) ) {
+		$t_select = array(
+			'id',
+			'name',
+			'real_name',
+			'email',
+			'access_level',
+			'language',
+			'timezone',
+			'created_at'
+		);
```

Comment:
The Command already defines the default fields, is it necessary to duplicate code here ?

Reply:
I have refactored the code to have a method that returns the default fields and used it from everywhere. The /users/me uses default fields and adds `projects` field to the list.